src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
You're extending an abstract class to define behavior for the Circle class. According to the checklist, abstract classes and interfaces have different use cases. Try to figure out when to use both in this task. If you're blocked, this link may give you a hint: https://stackoverflow.com/a/479168